README.md

```diff
@@ -154,23 +214,54 @@ The final bot provided is a QueryBot.
 This bot lets you query a collection of documents.
 To use it, you have two options:
 
-1. Pass in a list of paths to text files, or
-2. Pass in a session name of a previously instantiated `QueryBot` that model. (This will load the previously-computed text index into memory.)
+1. Pass in a list of paths to text files and make Llamabot create a new collection for them, or
+2. Pass in the `collection_name` of a previously instantiated `QueryBot` model. (This will load the previously-computed text index into memory.)
 
-As an illustrative example:
+An illustrative example creating a new collection:
 
 ```python
 from llamabot import QueryBot
 from pathlib import Path
 
-blog_index = Path("/path/to/index.json")
-bot = QueryBot(system_message="You are an expert on Eric Ma's blog.", session_name="eric_ma_blog")  # this loads my previously-embedded blog text.
-# alternatively:
-# bot = QueryBot(system_message="You are an expert on Eric Ma's blog.", session_name="eric_ma_blog", document_paths=[Path("/path/to/blog/post1.txt"), Path("/path/to/blog/post2.txt"), ...])
+bot = QueryBot(
+  system_prompt="You are an expert on Eric Ma's blog.", collection_name="eric_ma_blog",
```

Comment:
Could you split `collection_name` onto the next line please? Would look like this:

```suggestion
  system_prompt="You are an expert on Eric Ma's blog.", 
  collection_name="eric_ma_blog",
```

This is mostly for consistency.